src/panes.ts

```diff
@@ -108,8 +108,14 @@ export type LoginPane = Pane<
     credential?: "password" | "api_key"
     default_user_identifier?: string
     provider: ProviderMetadata
+    additional_fields?: AnyField["props"][]
```

Comment:
the criteria for "additional field" vs "regular field" is pretty nebulous, but also the way you have to commit to this repo is pretty annoying, so I think this escape hatch is just pragmatic right now

Reply:
we're trading type safety because the repo configuration is hefty I think

Reply:
i think i'll just use the fields pane. I think at this point this setup is getting out of hand, and it might require a new approach at some point, so i rather don't increase the complexity for now.